imap_processing/idex/idex_cdf_attrs.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+
+from imap_processing import common_cdf_attrs
+
+# Set IDEX software version here for now
+__version__ = "1"
```

Comment:
```suggestion
from imap_processing import common_cdf_attrs
from imap_processing.idex import version
```
Then in `imap_processing/idex/__init__.py`
```
version = "01"
```
I feel like we will use this variable a lot in different places. This way, it makes it easy to import anywhere. What do you think?